console/jsre/deps/web3.js

```diff
@@ -3850,6 +3850,11 @@ var inputBlockNumberFormatter = function (blockNumber) {
     return utils.toHex(blockNumber);
 };
 
+var inputEmptyFormatter = function (a) {
+  if (a === undefined) return "";
+  else                 return a;
+}
```

Comment:
After updating `web3.js`, you need to generate `bindata.go` again. Refer to `console/jsre/deps/deps.go`